aldryn_forms/models.py

```diff
@@ -17,6 +19,7 @@
 from .helpers import is_form_element
 
 
+USES_TREEBEARD_ORDERING = LooseVersion(cms.__version__) >= LooseVersion('3.1')
```

Comment:
As a compromise, instead of checking for something specific to treebeard, change the var name to VERSION31 or something.